web-animations-1/Overview.bs

```diff
@@ -4580,8 +4595,16 @@ dictionary OptionalEffectTiming {
     time taken to complete a single iteration of the <a>animation
     effect</a>.
 
-    In this level of this specification, the string value <code>auto</code>
-    is treated as the value zero for the purpose of timing model calculations
+    If the <a lt="timeline duration">duration</dfn> of the Animation's
+    <a>timeline</a> is a percentage, a non <code>auto</code> duration will
+    be treated as 0. Future levels of this specification are expected to
+    introduce support for scaling duration, iterations and start/end delays
+    to fill a percentage timeline duration.
+
+    The string value <code>auto</code> is treated as the
+    <a lt="timeline duration">duration</dfn> of the Animation's <a>timeline</a>
+    (zero if the timeline is null) divided by the <a>iteration count</a>
+    for the purpose of timing model calculations
```

Comment:
Isn't this a breaking change? Won't `auto' now resolve to infinity for cases where it used to resolve to 0?

Reply:
This is calculated to be 0 now.